packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3152,6 +3169,47 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     }
   }
 
+  async seekRow(inputString: string, isBackwards = false): Promise<void> {
+    const {
+      gotoValueSelectedColumn: selectedColumn,
+      gotoValueSelectedFilter,
+      lastSeekedRow,
+    } = this.state;
+    const { model } = this.props;
+    if (isIrisGridProxyModel(model)) {
+      const { table } = model;
+      if (table !== undefined && !TableUtils.isTreeTable(table)) {
+        if (selectedColumn.type === 'java.lang.String') {
+          // is string column,
+          const isContains = gotoValueSelectedFilter === 1;
+          const rowIndex = await table.seekRow(
+            isBackwards ? lastSeekedRow - 1 : lastSeekedRow + 1,
+            selectedColumn,
+            'String',
```

Comment:
I was incorrect about this value always being `'String'`. You'll want it to match the column type, so check if column `isStringType`, set as `String`, check `TableUtils.isNumberType`, set to `Number`. Otherwise it'll compare the value as a string, and you may get `95` matching `9500` for a number instead of just `95`.

Reply:
You'll also want to try/catch around this, as the `seekRow` function could throw an exception if there's an error. Ideally pass the error down to the Goto value and display it.

Reply:
Will also need to check for `BigDecimal` / `BigInteger` types, as those _will_ need to pass the value as string (a `Number` could get truncated).